backend/settings.py

```diff
@@ -85,8 +85,17 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'user-data',
+        'HOST': '127.0.0.1',
+        'PORT': 3306,
+        'USER': 'root',
+        'PASSWORD': 'lmaozedong',
```

Comment:
Is this password meant to be here?

Reply:
Yeah, this doesn't seem like the best practice so going to remove this portion just so we have the SQL files.